build/tasks/doc.js

```diff
@@ -1,14 +1,33 @@
 var gulp = require('gulp');
-var tools = require('aurelia-tools');
 var paths = require('../paths');
-var yuidoc = require('gulp-yuidoc');
+var typedoc = require("gulp-typedoc");
+var typedocExtractor = require("gulp-typedoc-extractor");
+var runSequence = require('run-sequence');
 
 gulp.task('doc-generate', function(){
-  return gulp.src(paths.source)
-    .pipe(yuidoc.parser(null, 'api.json'))
-    .pipe(gulp.dest(paths.doc));
+  return gulp.src([paths.output + '*.d.ts', paths.doc + '/core-js.d.ts', './jspm_packages/github/aurelia/*/*.d.ts'])
+    .pipe(typedoc({
+            target: "es6",
```

Comment:
Indentation looks off. It'd be great to use single quotes consistently (double quotes also appear in the `require`s above). It also looks like some tabs snuck in further down.